Nobar/Scenes/Search/Cell/RecentCollectionViewCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  RecentKeywordCollectionViewCell.swift
+//  Nobar
+//
+//  Created by 김수연 on 2022/07/12.
+//
+
+import UIKit
+
+import Then
+import SnapKit
+
+final class RecentCollectionViewCell: UICollectionViewCell {
+
+  private let keywordLabel = UILabel().then {
+    $0.textColor = .black
```

Comment:
여기도 저희 컬러 팔레트를 써줄수 있나용~~
```suggestion
    $0.textColor = Color.black.getColor()
```

Reply:
넵 !!